addons/website_slides/views/slide_slide_views.xml

```diff
@@ -368,5 +383,30 @@
                 </p>
             </field>
         </record>
+
+        <record id="slide_slide_action_report_graph" model="ir.actions.act_window.view">
```

Comment:
Should be ``slide_slide_action_report_view_graph`` (missing the 'view') to indicated this is an act_window view :) . To propagate to other new view definitions herebelow.